lib/assets/schemas/umm-cmn-json-schema.json

```diff
@@ -1498,7 +1500,7 @@
     },
     "SpatialCoverageTypeEnum": {
       "type": "string",
-      "enum": ["HORIZONTAL", "VERTICAL", "ORBITAL"]
+      "enum": ["HORIZONTAL", "VERTICAL", "ORBITAL", "HORIZONTAL_VERTICAL", "ORBITAL_VERTICAL"]
     },
     "CoordinateSystemEnum": {
       "type": "string",
```

Comment:
I just downloaded the schema and compared what we have in this PR.
comment for line 1541: we have `THREADS` (for the first two instances) and the version online has `THREDDS` (for all three instances)
comment for line 1553: we have ...`google-earch`... and the online version has `google-earth`

what we have in this file is also in the merged one.